### ISSUE TITLE
Incompatible with version 40.4.0

### ISSUE THREAD

**Issue by zaunermax** [pixel-saver/pixel-saver#231]
I tried altering the `metadata.json` but had no success. I cannot get the extension to work.

My gnome version is 40.4.0

**Comment by tallero** [comment]
It could be true I didn't add .4 subversion because I switched to 41 before .3 release. I'll add all subversions soon.

**Comment by mbacou** [comment]
I've tested versions 1.24 and 1.26 on Ubuntu 21.10 / Gnome 40.4. works no problem in a XOrg session, but freezes Wayland upon login. No relevant log message.

**Comment by tallero** [comment]
Have you installed the dependencies? XWayland is installed and running?
Latest version works flawlessly on GNOME Shell 41.5

Il giorno gio 7 apr 2022 alle ore 01:35 Mel. Bacou ***@***.***>
ha scritto:

**Comment by mbacou** [comment]
Yes XWayland is running. I've been using this extension without any problem since Ubuntu 18.04 (Wayland and Xorg) -- not sure why the recent upgrade to Ubuntu 21.10 is causing problem.
In general which version of the extension should I use, 1.24 or the latest?

FYI my current config:

```
$  gnome-extensions list
pixel-saver@deadalnix.me
user-theme@gnome-shell-extensions.gcampax.github.com
hide-universal-access@akiirui.github.io
weatherintheclock@JasonLG1979.github.io
sound-output-device-chooser@kgshank.net
ding@rastersoft.com
ubuntu-appindicators@ubuntu.com
ubuntu-dock@ubuntu.com

```

**Comment by zaunermax** [comment]
Could you maybe open a new issue please? :raised_hands: 

This issue is soley for the fact that the extension is not available for 40.4.0
